lte/gateway/python/integ_tests/defs.mk

```diff
@@ -135,6 +135,8 @@ s1aptests/test_attach_detach_dedicated_deactivation_timer_expiry.py \
 s1aptests/test_dedicated_bearer_activation_idle_mode.py \
 s1aptests/test_attach_detach_secondary_pdn.py \
 s1aptests/test_attach_detach_secondary_pdn_looped.py \
+s1aptests/test_concurrent_secondary_pdns.py \
+s1aptests/test_concurrent_secondary_pdn_reject.py \
```

Comment:
let's add these to NON_SANITY_TEST as sessiond has issues with concurrent standalone PDN requests for the same UE (not always but will make these tests flaky). After @themarwhal lands the fix, we can add them back to EXTENDED_TESTS.